javascript/javascript2/week1/js2w1.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>JS2 homework week1</title>
+</head>
+<body>
+   
+    <div id="input">
```

Comment:
Not the best name for "id". I think better will be "input-box" or "input-section".